### PR TITLE
Update the doc issue template

### PR DESCRIPTION
**Pull request by guardrex**
Addresses #31144 

Thanks @Porkechebure! ... I'm updating the issue template here. Rick will get back to you on your routing issue shortly.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [.github/ISSUE_TEMPLATE/doc-issue.md](https://github.com/dotnet/AspNetCore.Docs/blob/3d2968b7be858cfc4fd75759e0f3bf794ce24768/.github/ISSUE_TEMPLATE/doc-issue.md) | [Before you open an issue](https://review.learn.microsoft.com/en-us/aspnet/core/.github/ISSUE_TEMPLATE/doc-issue?branch=pr-en-us-31146) |

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
Should we add

image

File: .github/ISSUE_TEMPLATE/doc-issue.md [hunk 1]
```diff
@@ -8,12 +8,9 @@ about: Create an issue to help us improve
 If the issue is with an ASP.NET Core document:
 
 * Do **not** open a new issue using this form.
-* Open the issue with the **This page** button and form at the bottom of the document's page.
+* Open the issue using the **Open a documentation issue** link and feedback form at the bottom of the article.
```
Comment (Rick-Anderson) [inline .github/ISSUE_TEMPLATE/doc-issue.md:11]:
```suggestion
* Open the issue using the **Open a documentation issue** link and feedback form at the bottom of the article:
<img  alt="new issues" src="https://github.com/dotnet/AspNetCore.Docs/assets/3605364/13d3fae0-7960-4233-80e7-02fdaa65e61a">
```

**Review by Rick-Anderson — APPROVED** [review]

**Review by wadepickett — APPROVED** [review]
Better.  Looks good to me.

File: .github/ISSUE_TEMPLATE/doc-issue.md (hunk 1 above)
Comment (guardrex) [reply .github/ISSUE_TEMPLATE/doc-issue.md:11]: It won't show up in the GH issue UI. Text only.

**Comment by guardrex** [comment]
I'll go ahead with it. Tom will patch it if there's something better to do with the text.

**Pull request merged**